### ISSUE TITLE
Functions with Tuple Return Types not Correctly Analyzed or Ignored

### ISSUE THREAD

**Issue by SeanMcTex** [peripheryapp/periphery#386]
# Reproduction steps:

1. Run periphery against this code:

```
import Foundation

public struct Foo {
    static func cheese() -> (Int, Int) {
        return (5, 10)
    }
}

public class Bar {
    // periphery:ignore
    let (i, j) = Foo.cheese()

    // periphery:ignore
    let k = 1029
}
```

# Expected:

No warnings, since `cheese()` is called and the `let` statements are each preceded with an ignore comment command.

# Actual:

```
/Users/sharky/Source/PeripheryTest/PeripheryTest/PublicTest.swift:11:17: warning: Function 'cheese()' is unused
/Users/sharky/Source/PeripheryTest/PeripheryTest/PublicTest.swift:18:10: warning: Property 'i' is unused
/Users/sharky/Source/PeripheryTest/PeripheryTest/PublicTest.swift:18:13: warning: Property 'j' is unused
```

**Comment by ileitch** [comment]
Fixed in 2.8.1.

**Comment by SeanMcTex** [comment]
Wow, that was some staggeringly quick turnaround. Thank you, Ian!
